src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -952,6 +955,7 @@ static TableParams createTableParamsFromRow(UntypedResultSet.Row row)
                           .comment(row.getString("comment"))
                           .compaction(CompactionParams.fromMap(row.getFrozenTextMap("compaction")))
                           .compression(CompressionParams.fromMap(row.getFrozenTextMap("compression")))
+                          .memtable(MemtableParams.get(row.getString("memtable")))
```

Comment:
On upgrade, where we don't have a value for "memtable", the idea is that we'll handle the null from this cell and get the default configuration, correct?